### PR TITLE
malli describe fixes

### PR DESCRIPTION
**Pull request by escherize**
- use titles for everything
- repeat "sounds" right
- biding time to re-merge upstream

### CODE REVIEW

File: src/metabase/util/malli/describe.clj [hunk 1]
```diff
@@ -192,11 +206,24 @@
...
+
+(defmethod accept :alt [_ _ children _] (str/join ", or " children))
+(defmethod accept :altn [_ _ children _] (str/join ", or " (tagged children)))
+
+(defmethod accept :+ [_ _ children _] (str "one or more " (str/join ", " children)))
+(defmethod accept :* [_ _ children _] (str "zero or more " (str/join ", " children)))
+(defmethod accept :? [_ _ children _] (str "zero or one " (str/join ", " children)))
+
+(defmethod accept :repeat [_ schema children _]
+  (str "repeat " (diamond (first children)) (repeat-suffix schema)))
```
Comment (escherize) [inline src/metabase/util/malli/describe.clj:226]: these were missing

File: src/metabase/util/malli/describe.clj [hunk 2]
```diff
@@ -116,36 +130,36 @@
...
-(defmethod accept 'int? [_ schema _ _] (str "integer" (min-max-suffix-number schema)))
-(defmethod accept :int [_ schema _ _] (str "integer" (min-max-suffix-number schema)))
+(defmethod accept 'integer? [_ schema _ _] (str "integer" (titled schema) (min-max-suffix-number schema)))
+(defmethod accept 'int? [_ schema _ _] (str "integer" (titled schema) (min-max-suffix-number schema)))
+(defmethod accept :int [_ schema _ _] (str "integer" (titled schema) (min-max-suffix-number schema)))
 
-(defmethod accept 'double? [_ schema _ _] (str "double" (min-max-suffix-number schema)))
-(defmethod accept :double [_ schema _ _] (str "double" (min-max-suffix-number schema)))
+(defmethod accept 'double? [_ schema _ _] (str "double" (titled schema) (min-max-suffix-number schema)))
+(defmethod accept :double [_ schema _ _] (str "double" (titled schema) (min-max-suffix-number schema)))
```
Comment (escherize) [inline src/metabase/util/malli/describe.clj:162]: all of these can be titled!

File: test/metabase/util/malli/describe_test.clj [hunk 3]
```diff
@@ -1,97 +1,103 @@
...
+    (is (= "integer between 0 and 1 inclusive"
+           (umd/describe [:int {:min 0 :max 1}]))))
+
+  (testing "repeat"
+    (is (= "repeat <integer> at least 1 time"
+           (umd/describe [:repeat {:min 1} int?])))
+    (is (= "repeat <integer> at most 7 times"
+           (umd/describe [:repeat {:max 7} int?])))
+    (is (= "repeat <integer> at least 1 time, up to 7 times"
+           (umd/describe [:repeat {:min 1 :max 7} int?])))))
```
Comment (escherize) [inline test/metabase/util/malli/describe_test.clj:103]: repeat was quite bad, now it's good.

File: src/metabase/util/malli/describe.clj [hunk 4]
```diff
@@ -29,6 +29,20 @@
       max (str " with length >= " max)
       :else "")))
 
+(defn pluralize-times [n]
+  (when n
+    (if (= 1 n) "time" "times")))
+
+(defn- repeat-suffix [schema]
```
Comment (qnkhuat) [inline src/metabase/util/malli/describe.clj:36]: not related but I wonder if it's better to write these suffixes with defmethod? will makes adding new stuff easier.

File: src/metabase/util/malli/describe.clj [hunk 5]
```diff
@@ -209,10 +236,12 @@
         additional-properties (:closed (mc/properties schema))
         kv-description (str/join ", " (map (fn [[k _ s]] (str k (when (contains? optional  k) " (optional)") " -> " (diamond s))) children))]
     (str/trim
-     (cond-> (str "map ")
+     (cond-> (str "map " (titled schema))
        (seq kv-description) (str "where {" kv-description "} ")
        additional-properties (str "with no other keys ")))))
 
+(describe [:map {:title "X" :closed true} [:a int?]])
```
Comment (qnkhuat) [inline src/metabase/util/malli/describe.clj:243]: 🧹

**Review by qnkhuat — APPROVED** [review]
❤️

File: src/metabase/util/malli/describe.clj (hunk 5 above)
Comment (escherize) [reply src/metabase/util/malli/describe.clj:243]: 😅

```suggestion
```

File: src/metabase/util/malli/describe.clj (hunk 4 above)
Comment (escherize) [reply src/metabase/util/malli/describe.clj:36]: That's probably worth it. There might not be any more suffixes to add 🤞 -- If I touch this file again I'll do that

**Pull request merged**